src/templates/pages/example/modal-confirmation/code.js

```diff
@@ -2,7 +2,7 @@ import { h, Fragment } from 'preact';
 
 const Code = () => <Fragment>
     <button type="button" class="modal-confirmation__btn js-modal-confirmation__btn">Delete</button>
-    <div id="modal-confirmation" role="region" class="js-modal-confirmation modal-confirmation modal-container" data-modal-toggle="js-modal-confirmation__btn">
+    <div id="modal-confirmation" aria-labelledby="modal-label" role="region" class="js-modal-confirmation modal-confirmation modal-container" data-modal-toggle="js-modal-confirmation__btn">

```

Comment:
Not sure about these ones but I think we need to test them. Let's get Susannah to take a look.